content/guides/userguides/consumer-vignettes/export-to-terra.md

```diff
@@ -6,115 +6,146 @@ subTitle: ""
-
-After registering, you can view multiple workspaces dedicated to using HCA data, such as the:
+### HCA data-related tutorial workspaces
+In addition to running the tutorial below, registering for Terra allows you to accesss multiple tutorial workspaces dedicated to using HCA data as described below:
 
 * [Optimus workspace](https://app.terra.bio/#workspaces/featured-workspaces-hca/HCA_Optimus_Pipeline) for processing 10x data with the [Optimus Pipeline](/pipelines/optimus-workflow).
 * [Smart-seq2 workspace](https://app.terra.bio/#workspaces/featured-workspaces-hca/HCA%20Smart-seq2%20Multi%20Sample%20Pipeline) for processing Smart-seq2 data with the [Smart-seq2 Multi-Sample Pipeline](/pipelines/smart-seq2-workflow).
 * [Intro-to-HCA-data-on-Terra workspace](https://app.terra.bio/#workspaces/featured-workspaces-hca/Intro-to-HCA-data-on-Terra) for exporting HCA data and analyzing it with community tools like [Seurat](https://satijalab.org/seurat/index.html), [Scanpy](https://scanpy-tutorials.readthedocs.io/en/latest/index.html), [Cumulus](https://cumulus.readthedocs.io/en/latest/index.html), and [Pegasus](https://pegasus.readthedocs.io/en/stable/#). 
 
-## Overview of Dockstore
+### Video demonstration of importing HCA data to a Terra workspace
+Prior to trying the tutorial, you might want to watch this very brief [video demonstration](https://www.youtube.com/watch?v=G3N2i3NjsfY) of importing HCA data into Terra and using the Intro-to-HCA-data-on-Terra workspace. This video is additionally available in this [Terra blog](https://terra.bio/discover-how-to-use-human-cell-atlas-data-in-terra/) about the workspace.
 
-[Dockstore](https://dockstore.org/) is a platform for sharing bioscience tools by wrapping them in Docker containers and describing their use with high-level workflow languages like the Common Workflow Language (CWL) and the Workflow Description Language (WDL).
 
-For more information about how to use the Dockstore, see the [Dockstore documentation](https://docs.dockstore.org/en/develop/).
+## Tutorial — Automated processing of HCA BAM files in Terra 
 
-## Step one: finding BAM files with the HCA Data Explorer
+This tutorial guides you through finding and setting up a simple example workflow in Terra that you can use to process BAM files exported from the HCA Data Explorer.
 
-You can use the <link-to-browser relativelink="/projects">HCA Data Explorer</link-to-browser> to find data to export to Terra.
-The Data Explorer lists projects with data available for download from the Data Store and lets you filter the data for a number of attributes.
+For an additional Terra tutorial on using community tools with HCA data, see the [Intro-to-HCA-data-on-Terra workspace](https://app.terra.bio/#workspaces/featured-workspaces-hca/Intro-to-HCA-data-on-Terra). 
```

Comment:
I might put this at the end of the tutorial as an extra step to try if you want more practice kind of thing.

Reply:
Thanks for the suggestion. I added it to the end as well. I put it here too because I wanted them to see it before they started on this tutorial in case they are more interested in community tools than using the MD5sum workflow.